R/expr-expr.R

```diff
@@ -1262,7 +1261,7 @@ expr__dot <- function(other) {
 #' df$select(pl$col("foo")$reshape(c(3, -1)))
 expr__reshape <- function(dimensions) {
   wrap({
-    if (is.numeric(dimensions) && anyNA(dimensions)) {
+    if (anyNA(dimensions)) {
```

Comment:
Caused the "unreachable" message that was saved in the snapshot

Reply:
I think we should check the type before `anyNA()`.